executor/index_lookup_join.go

```diff
@@ -641,6 +642,9 @@ func (iw *innerWorker) fetchInnerResults(ctx context.Context, task *lookUpJoinTa
 }
 
 func (iw *innerWorker) buildLookUpMap(task *lookUpJoinTask) error {
+	if task.innerResult == nil {
```

Comment:
Since `ctx.Err()` returned when receiving signal from `ctx.Done`, this change is not needed anymore.